docs/cluster-configuration.md

```diff
@@ -1117,6 +1123,35 @@ With the `Custom` profile, the cipher list should be expressed according to Open
 
 On plain k8s, where APIServer CR is not available, the default value will be `Intermediate`.
 
+## configure Applications Aware Quota (AAQ)
+To enable the AAQ feature, set the [enableApplicationsAwareQuota feature gate](#enableapplicationsawarequota-feature-gate)
+
+To configure AAQ, add the `applicationAwareConfig` object to the HyperConverged resource's spec. The 
+`applicationAwareConfig` object contains several fields:
+* `vmiCalcConfigName` - determine how resource allocation will be done with ApplicationsResourceQuota.
+  allowed values are:
+  * `VmiPodUsage` - Calculate usage of launcher like any other pod but hide migration additional resources
+  * `VirtualResources` - Calculate usage of launcher like any other pod but hide migration additional resources
+  * `DedicatedVirtualResources` (default) - Calculate vmi.requests.memory as the vmi's ram size and vmi.requests.cpu as 
+    number of threads of vmi in this configuration no memory.request/limits and cpu.request/limits won't be included
+  * `IgnoreVmiCalculator` - in this configuration no memory.request/limits and cpu.request/limits won't be included
+* `namespaceSelector` - determines in which namespaces scheduling gate will be added to pods. This field is a standard 
+                        kubernetes selector.
+* `enableClusterAppsResourceQuota` (default = false) - set to true, to allows creation and management of ClusterAppsResourceQuota
+  
+  **note**: this setting cause some performance cost. Only set to true if there is a good reason.
```

Comment:
VmiPodUsage calculates pod usage for VM-associated pods while concealing migration-specific resources.

VirtualResources allocates resources for VM-associated pods, using the VM's RAM size for memory and CPU threads for processing.

DedicatedVirtualResources (default) allocates resources for VM-associated pods, appending a /vm suffix to requests/limits.cpu and requests/limits.memory, derived from the VM's RAM size and CPU threads. Notably, it does not allocate resources for the standard requests/limits.cpu and requests/limits.memory .

IgnoreVmiCalculator avoids allocating VM-associated pods differently from normal pods, maintaining uniform resource allocation.